build/controllers/PhpDocController.php

```
@@ -847,33 +847,41 @@ protected function generateClassPropertyDocs($fileName)
                     if (!$this->hasSetterInParents($className, $propName)) {
-                        $docLine .= '-read';
+                        $annotationSuffix = '-read';
                     }
                 } elseif (isset($prop['set'])) {
                     if (!$this->hasGetterInParents($className, $propName)) {
-                        $docLine .= '-write';
+                        $annotationSuffix = '-write';
                     }
                 } else {
                     continue;
                 }
```

PHPMD (2.15.0)

[error] The method generateClassPropertyDocs uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)